package.json

```diff
@@ -120,5 +122,11 @@
       "babel-runtime",
       "jscodeshift"
     ]
-  }
+  },
+  "bundlesize": [
+    {
+      "path": "./dist/preact.min.js",
+      "threshold": "4Kb"
```

Comment:
I think "threshold" is the amount of bytes the build is allowed to change in a given PR/commit, so for Preact we can set this super low since the size isn't allowed to increase:  maybe something like 100b.